resolvconf/resolvconf.go

```diff
@@ -89,7 +93,7 @@ func GetLastModified() ([]byte, string) {
 //    code will remove all IPv6 nameservers if it is not enabled for containers
 //
 // It returns a boolean to notify the caller if changes were made at all
-func FilterResolvDNS(resolvConf []byte, ipv6Enabled bool) ([]byte, bool) {
+func FilterResolvDNS(resolvConf []byte, ipv6Enabled bool) ([]byte, string, bool, error) {
```

Comment:
Let's also remove the `changed` bool since it's not of much use and I don't see how any one would need that